examples/nemo/scripts/acc_kernels_trans.py

```diff
@@ -387,14 +447,13 @@ def trans(psyir):
         if subroutine.name.lower() not in ACC_IGNORE:
             print(f"Transforming {subroutine.name} with acc kernels")
             enhance_tree_information(subroutine)
-            inline_calls(subroutine)
-            have_kernels = add_kernels(subroutine.children)
-            if have_kernels and ACC_EXPLICIT_MEM_MANAGEMENT:
-                print(f"Transforming {subroutine.name} with acc enter data")
-                ACC_EDATA_TRANS.apply(subroutine)
+            # inline_calls(subroutine)
```

Comment:
Please add a comment here saying something like "Inlining isn't robust enough for use yet"